portalnet/src/discovery.rs

```diff
@@ -111,28 +111,33 @@ impl Discovery {
                 .map_err(|e| format!("When adding key to servers ENR: {e:?}"))?
         };
 
-        // Check if we have an old version of our Enr and if we do, increase our sequence number
-        if let Some(enr_file_location) = portal_config.enr_file_location {
-            let trin_enr_file_location = enr_file_location.join(ENR_FILE_NAME);
+        // This should only never run in test cases.
+        if portal_config.trin_data_dir != PathBuf::default() {
+            // Check if we have an old version of our Enr and if we do, increase our sequence number
+            let trin_enr_file_location = portal_config.trin_data_dir.join(ENR_FILE_NAME);
```

Comment:
I would change to `trin_enr_path` - basically, `_path` suffix to these types of variable names describing a file path is what we typically do in this codebase, and I think we should continue with that convention to maintain consistency
 - https://github.com/ethereum/trin/blob/040ad41f66b17733d2e7c2a07b6b07644dac8225/portal-bridge/src/cli.rs#L142
 - https://github.com/ethereum/trin/blob/040ad41f66b17733d2e7c2a07b6b07644dac8225/portal-bridge/src/bridge.rs#L50
 - https://github.com/ethereum/trin/blob/040ad41f66b17733d2e7c2a07b6b07644dac8225/rpc/src/lib.rs#L74